test/testes.cpp

```diff
@@ -0,0 +1,166 @@
+/* VideoParser
```

Comment:
One question, what's the difference between this test and the test.ccp that already exists?

Reply:
This test does not need demuxeres and get only elementary stream as inputs. Its not possible to have demuxeres and GstVkVideoParser to be fully static. And I was not able to include demuxeres in GstVkVideoParser as the symbols from demuxeres were not exported. That's why I disabled the tests in full static mode and kept one test: testes.cpp